src/Symfony/Component/Yaml/Inline.php

```diff
@@ -591,6 +591,7 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                     case 0 === strpos($scalar, '!!binary '):
                         return self::evaluateBinaryScalar(substr($scalar, 9));
                     case preg_match('/^(-|\+)?[0-9][0-9,]*(\.[0-9_]+)?$/', $scalar):
+                        @trigger_error('Using the comma as a group separator for floats is deprecated since version 3.2 and will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
@Taluu but it's true what @xabbuh said: this is handled in the code above this `case:` and there are tests about this and they pass.